feature_engine/base_transformers.py

```diff
@@ -154,17 +157,26 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
 
         return X
 
-    def get_feature_names_out(self) -> List:
+    @Substitution(
+        input_features=_input_features_docstring,
+    )
+    def get_feature_names_out(
+            self, input_features: Optional[Union[List, str]] = None
+    ) -> List:
         """Get output feature names for transformation.
 
+        Parameters
+        ----------
+        {input_features}
```

Comment:
I think we could make the entire docstring (not just input_features) part of the `_input_features_docstring` docstring in the docstrings module. It is the same everywhere I think.